root/static/js/canto-modules.js

```diff
@@ -3571,8 +3571,11 @@ canto.directive('genotypeEdit',
 
 var genotypeViewCtrl =
   function($scope, CantoGlobals, CursSettings) {
-    $scope.init = function(annotationCount) {
+    var ctrl = this;
+
+    $scope.init = function(annotationCount, organismType) {
       $scope.annotationCount = annotationCount;
+      ctrl.organismType = organismType;
```

Comment:
Yes, but does it need to be on `$scope` if it's never used by the template? I was under the impression that `$scope` properties are like public properties, so I wasn't sure if there was any need to declare it public when it's only used by the controller.

Reply:
Yep, I think you're right.  I'm nervous about `this` since there's no guarantee that it's the same `this` when a method gets called.  But it's no problem in this case because you're capturing `this` as `ctrl`.